tests/Doctrine/Tests/Common/Cache/PhpFileCacheTest.php

```diff
@@ -66,6 +66,18 @@ protected function _getCacheDriver()
     {
         return new PhpFileCache($this->directory);
     }
+
+    public function testFailureOnIncludeNotSettedCacheDoesntGetCaughtAsError()
```

Comment:
You could add this test to the generic CacheTest so it is run against every cache driver. If doctrine decides to support custom error handlers that do not respect the `error_reporting`, then it applies to all drivers and not just the PhpFileCache.